packages/tracing/src/index.bundle.replay.ts

```diff
@@ -0,0 +1,7 @@
+import { Replay } from '@sentry/browser';
+
+import * as Sentry from './index.bundle';
+
+Sentry.Integrations.Replay = Replay;
```

Comment:
It was the easiest/cleanest way I could find to handle this in a dedicated input file. Otherwise it gets quite messy when trying to do this inside of `index.bundle.ts` 😬